newdle/client/src/answerSelectors.js

```diff
@@ -19,11 +20,17 @@ export const getCalendarDates = createSelector(
 );
 export const getActiveDate = state =>
   state.answer.calendarActiveDate || getCalendarDates(state)[0] || serializeDate(moment());
+/**
+ * All "busy" times, indexed by day
+ */
 export const getBusyTimes = () => ({
   '2019-10-03': [['12:00', '13:00'], ['13:30', '14:30']],
   '2019-10-11': [['09:45', '12:00'], ['16:00', '18:00']],
   '2019-10-12': [['22:00', '00:00']],
 });
+/**
```

Comment:
Is `/** blah blah */` valid for jsdoc-style comments that fit in a single line?

Reply:
Yeah, it's just my editor which adds multi-line by default. I'll fix it.